app/src/main/java/com/victor/adasfeed/ui/recyclerview/adapter/PostAdapter.kt

```diff
@@ -53,32 +33,44 @@ class PostAdapter(private val postList: MutableList<Post> = Faker.makePostList()
     override fun getItemCount() = postList.size
 
     override fun onBindViewHolder(holder: PostViewHolder, position: Int) {
-        holder.bind(postList, position)
-    }
+        holder.apply {
+            val p = postList[position]
 
-    fun addNewPost(newPost: Post) {
-        postList.add(newPost)
-        notifyItemInserted(postList.indexOf(newPost))
-    }
-}
+            tvUsername.text = p.userName
+            tvDescription.text = p.description
+            ivPost.setImageResource(p.imagePost)
+            ivUser.setImageResource(p.imageUser)
 
-class PostDiffUtilCallback(
-    private val oldPostList: List<Post>,
-    private val newPostList: List<Post>,
-) : DiffUtil.Callback() {
-    override fun getOldListSize() = oldPostList.size
+            ivHeart.setOnClickListener {
+                ivHeart.setImageResource(R.drawable.favorite_filled)
+            }
+
+            ivPost.setOnLongClickListener {
+                removePost(position)
+
+                Snackbar.make(it, "Post deletado", 5000).apply{
```

Comment:
nit: you can add the strings to the resources folder and `5000` to a constant to be more readable